packages/client/lib/service/txpool.ts

```diff
@@ -718,8 +751,41 @@ export class TxPool {
   }
 
   _logPoolStats() {
+    let broadcasts = 0
+    let broadcasterrors = 0
+    let knownpeers = 0
+    for (const sendobjects of this.knownByPeer.values()) {
```

Comment:
Should we contepmplate putting this whole method behind a debug flag or something?  I'm just wondering since this is a sort of a lot of iterating over arrays just report broadcast stats.  Maybe at a minimum, we could add a check [here](https://github.com/ethereumjs/ethereumjs-monorepo/blob/6c429278713ab6447b98ac236e02854de32f131e/packages/client/lib/service/txpool.ts#L190) to only turn this interval on if the loglevel is set to `info` or higher?  These logs kinda feel superfluous and I wouldn't want to run them by default.  So, we could maybe even turn the log level for these messages to debug?  I dunno, just a thought.

Reply:
hmmm i can understand the reasoning, i would be ok to put them onto a debug level functioanlity where we can switch on/off module debug levels with the key press without requiring restarts, 
would it be possible to enhance `debug` with this?